### ISSUE TITLE
Can we do https with apiMocker?

### ISSUE THREAD

**Issue by httpete** [gstroup/apimocker#31]

**Comment by gstroup** [comment]
No, not at the moment.  It looks like that would not be too difficult to add in, but then we'd have to deal with certificates.  Is this something you need?

**Comment by httpete** [comment]
Hi, I am working around this by doing my own reverse proxy with http-proxy that simply proxies the https to the http that api mocker supports.What would be perfect is just as grunt-contrib-connect config does, allowing a { protocol: 'https' } and it just works over https. I bet we could take what they did inside contrib-connect and take it.

**Comment by gstroup** [comment]
Interesting... I'll take a look at that option in grunt-contrib-connect.  I usually just do my local dev with http, and worry about https on a shared server.  But I can see that it would be nice to have in apimocker.

**Comment by httpete** [comment]
I was exactly the same way - until a new requirement forced https in my local env on me. In a way it's good, we are more in parallel with the production env.

**Comment by raagav-iyer** [comment]
Also would like to consider --proxy url to also support https.
Any timelines on having https support. Also any guidance to implement https support is also much appreciated, I can contribute back.
